wordpress-mobile/WordPress-iOS
# Merge Login and Signup Epilogues

Ref #13413, #13939

This is to merge the Epilogue feature branch into `develop`. It encompasses all changes for #13413 and #13939. 

---
Known Issues:
- Status bar text is white in light mode.
- Password field: when the view refreshes (i.e. changing split mode, toggling light/dark mode), the field is inset by a few points.
- Password field: Secure text icon color has not been updated.

---
To test:
- Login.
  - Verify the new epilogue appears as expected.
- Signup.
  - Verify the new epilogue appears as expected.

| Login | Signup |
|--------|-------|
| ![login](https://user-images.githubusercontent.com/1816888/80739756-35e32580-8ad4-11ea-95a1-bca7990e60b4.png) | ![signup](https://user-images.githubusercontent.com/1816888/80739772-3c719d00-8ad4-11ea-9e15-449fe16c76b0.png) |

---
To show the Signup Epilogue without actually signing up:

- In WPiOS `Podfile`, point `WordPressAuthenticator` to your local Auth.
- In `LoginViewController`, change `showLoginEpilogue` to show the signup epilogue:

```
func showLoginEpilogue(for credentials: AuthenticatorCredentials) {
    guard let navigationController = navigationController else {
        fatalError()
    }
    
    showSignupEpilogue(for: credentials)
    
//        authenticationDelegate.presentLoginEpilogue(in: navigationController, for: credentials) { [weak self] in
//            self?.dismissBlock?(false)
//        }
}
```

- _Login_ with an existing account, and the _signup_ epilogue will be displayed.

---
PR submission checklist:
